airflow/config_templates/default_airflow.cfg

```diff
@@ -33,6 +33,10 @@
 # subfolder in a code repository. This path must be absolute.
 dags_folder = {AIRFLOW_HOME}/dags
 
+# Replaces the dags folder path for incomming commands to always use the 
```

Comment:
Just a note to remove that one when/if we fix the location to be relative to AIRFLOW_DAG